cross-chain/sei/scripts/configure-ntt-paths.ts

```diff
@@ -0,0 +1,563 @@
+const PROXY_ABI = [
+  "function implementation() external view returns (address)",
+  "function admin() external view returns (address)",
+]
+
+// Note: Transceiver path configuration is handled automatically by NTT Manager
+// The Transceiver interface doesn't have peer registration - it routes based on
+// the recipientNttManagerAddress passed from the NTT Manager
+
+/**
+ * Convert address to bytes32 format (pad to 32 bytes)
+ */
+function addressToBytes32(address: string, ethersUtils: any): string {
+  return ethersUtils.hexZeroPad(address, 32)
+}
+
+/**
+ * Configure NTT Manager peer
+ */
+async function configureNttManagerPeer(
+  nttManager: any,
+  chainId: number,
+  peerAddress: string,
+  decimals: number,
+  inboundLimit: any,
+  signer: any,
+  ethersUtils: any
+): Promise<boolean> {
+  try {
+    const peerBytes32 = addressToBytes32(peerAddress, ethersUtils)
+
+    // Check current peer
+    try {
+      const currentPeer = await nttManager.getPeer(chainId)
+      if (currentPeer.toLowerCase() === peerBytes32.toLowerCase()) {
+        console.log(`      ✅ NTT Manager peer already configured correctly`)
+        return true
+      }
+      console.log(`      📝 Updating NTT Manager peer from ${currentPeer} to ${peerBytes32}`)
+    } catch (error) {
+      console.log(`      📝 Setting NTT Manager peer (getPeer not available, proceeding)`)
+    }
+
+    // Set peer with explicit gas limit
+    // For proxy contracts, we call through the proxy which forwards to implementation
+    console.log(`      📝 Sending setPeer transaction...`)
+    console.log(`         Chain ID: ${chainId}`)
+    console.log(`         Peer Address (bytes32): ${peerBytes32}`)
+    console.log(`         Decimals: ${decimals}`)
+    console.log(`         Inbound Limit: ${ethersUtils.formatEther(inboundLimit)} tBTC`)
+    
+    // Try to estimate gas first to see if it would succeed
+    try {
+      const gasEstimate = await nttManager.estimateGas.setPeer(chainId, peerBytes32, decimals, inboundLimit)
+      console.log(`         Gas estimate: ${gasEstimate.toString()}`)
+    } catch (estimateError: any) {
+      console.warn(`         ⚠️  Gas estimation failed: ${estimateError.message}`)
+      console.warn(`         This may indicate the transaction will revert`)
+      if (estimateError.data) {
+        console.warn(`         Error data: ${estimateError.data}`)
+      }
+    }
+    
+    // Try with callStatic first to get the actual revert reason
+    try {
+      await nttManager.callStatic.setPeer(chainId, peerBytes32, decimals, inboundLimit)
+    } catch (staticError: any) {
+      console.error(`      ❌ Transaction would revert: ${staticError.message}`)
+      if (staticError.data && staticError.data !== "0x") {
+        console.error(`      Error data: ${staticError.data}`)
+        // Try to decode common error selectors
+        const errorSelectors: Record<string, string> = {
+          "0x8da5cb5b": "Ownable: caller is not the owner",
```

Comment:
Could you explain how this mapping works? I can see that `0x8da5cb5b` decodes to `owner()`:

```
$ cast sig "owner()"
0x8da5cb5b
```